### DISCUSSION TITLE
account question

### DISCUSSION THREAD

**Discussion by saitoh183** [arabcoders/watchstate#349] [General] [answered]
I wanted to know if we can also transfer watched status of non plex home users? When it ask me which account to choose, i only see my plex home users. What about the other accounts. Not managed accounts.

**Comment by arabcoders** [comment, marked as answer]
> I wanted to know if we can also transfer watched status of non plex home users? When it ask me which account to choose, i only see my plex home users. What about the other accounts. Not managed accounts.

Hello, Due to the way plex create access tokens you gonna need that friend access token to generate one for the server, You don't need their admin one just the one that can access your server to pull info from. We cannot generate tokens for invited guests/friends from your own account.

**Reply by saitoh183** [reply]
ah ok thanks!
